### PR TITLE
Add TfsCmdlets

### PR DESCRIPTION
**Pull request by igoravl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23646)

### CODE REVIEW

**Review by ImJoakim — CHANGES_REQUESTED** [review]
You seem to have added the manifest twice, onc in the manifests folder which is correct, and one which is not inside the manifests folder. Do the following:

Remove the following files:
```
i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.installer.yaml
i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.locale.en-US.yaml
i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.yaml
```

Keep the following files:
```
manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.installer.yaml
manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.locale.en-US.yaml
manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.yaml
```

**Review by ImJoakim — COMMENTED** [review]
Also, the version needs to match the ARP (Add / Remove Programs) which in this case is `2.0.1.2474`:

image

File: manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Igoravl.TfsCmdlets
+PackageVersion: 2.0.1
```
Comment (ImJoakim) [inline manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.1.2474
```

File: manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Igoravl.TfsCmdlets
+PackageVersion: 2.0.1
```
Comment (ImJoakim) [inline manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.1.2474
```

File: manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Igoravl.TfsCmdlets
+PackageVersion: 2.0.1
```
Comment (ImJoakim) [inline manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.yaml:5]:
```suggestion
PackageVersion: 2.0.1.2474
```

**Comment by ImJoakim** [comment]
Remember to change the version folder as well:

`manifests/i/Igoravl/TfsCmdlets/2.0.1/` -> `manifests/i/Igoravl/TfsCmdlets/2.0.1.2474/`

File: manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Tags:
+- TfsCmdlets
+- TFS
+- VSTS
+- PowerShell
+- Azure
+- AzureDevOps
+- DevOps
+- ALM
+- TeamFoundationServer
```
Comment (ImJoakim) [inline manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.locale.en-US.yaml:30]: Also, best practice is to use all lower case tags. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
Tags:
- tfscmdlets
- tfs
- vsts
- powershell
- azure
- azuredevops
- devops
- alm
- teamfoundationserver
```

**Comment by ImJoakim** [comment]
> Hello @igoravl,
> The package manager bot determined that the metadata was not compliant.
> 
> Please verify the manifest file is compliant with the package manager [1.0 manifest specification](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md).
> Make sure the ID is of the form publisher.appname and that the folder structure is manifests\partition\publisher\appname\version.
> Note: The path and "PackageIdentifier" are case sensitive.
> Be sure to use a tool like VSCode (https://code.visualstudio.com/) to make sure the manifest YAML syntax is correct.
> 
> You could also try our [Windows Package Manager Manifest Creator Preview](https://github.com/microsoft/winget-create).
> 
> For details on the specific error, see the details link below in the build pipeline.

See https://github.com/microsoft/winget-pkgs/pull/23646#issuecomment-892136597

**Review by ImJoakim — CHANGES_REQUESTED** [review]
Seems like this file was just added. This needs to be removed:

`manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.installer.yaml`

File: manifests/i/Igoravl/TfsCmdlets/2.0.1.2474/Igoravl.TfsCmdlets.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: user
+  InstallerUrl: https://github.com/igoravl/TfsCmdlets/releases/download/v2.0.1%2B2474.3/TfsCmdlets-2.0.1.msi
+  InstallerSha256: 50EB556631C029514DF3838E58256C742C5B513EEAEB87DE44E4B85E2FB0349A
+  InstallerSwitches:
+    Silent: /qn
+    SilentWithProgress: /qb
+  ProductCode: 'bade96bc-5a7f-4dd3-9610-33fd76e0c9da'
```
Comment (ImJoakim) [inline manifests/i/Igoravl/TfsCmdlets/2.0.1.2474/Igoravl.TfsCmdlets.installer.yaml:21]:
```
PS C:\Users\Joakim\Downloads> Get-MsiProductCode.ps1 .\TfsCmdlets-2.0.1.msi
{72643634-0E33-4121-87F4-3CFD86B90884}
```

```suggestion
  ProductCode: '{72643634-0E33-4121-87F4-3CFD86B90884}'
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**